src/dashboard/src/components/helpers.py

```diff
@@ -304,14 +304,24 @@ def processing_config_path():
     )
 
 
-def stream_file_from_storage_service(url, error_message='Remote URL returned {}'):
-    stream = requests.get(url, stream=True, timeout=django_settings.STORAGE_SERVICE_CLIENT_TIMEOUT)
+def stream_file_from_storage_service(url,
+                                     error_message='Remote URL returned {}',
+                                     preview_file=False):
+    # Repetetive or constant values to use below when seeting the headers.
+    header_content_disposition = 'Content-Disposition'
+    content_disposition_inline = 'inline'
+    storage_timeout = django_settings.STORAGE_SERVICE_CLIENT_TIMEOUT
+    stream = requests.get(url,
```

Comment:
A note for the future about this function (not for stable): it looks like the caller expects at all times that the func returns a valid response. If this is really the case I think that we should run get() inside a `try/except` combined with `raise_for_status()`. This way we could handle both `ConnectionError` (connectivity issues) and `HTTPError` (status_code isn't 2xx).